qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -140,7 +140,8 @@ def _swap_condition(property_set):
         _unroll = [Unroller(basis_gates)]
     elif translation_method == 'translator':
         from qiskit.circuit.equivalence_library import SessionEquivalenceLibrary as sel
-        _unroll = [UnrollCustomDefinitions(sel, basis_gates),
+        _unroll = [UnitarySynthesis(basis_gates),
```

Comment:
does UnitarySynthesis need to be here given that it is part of `_opt` too? (_unroll passes are called after _opt passes).

Reply:
Good catch, this is redundant here.